nbs/getting_started.ipynb

```diff
@@ -85,7 +85,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Why did you make this change? I though quarto auto-updates suffixes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/950/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>It didn’t seem to modify the URL at all for README.md, resulting in a broken link. Good catch - maybe there’s a deeper issue there?</p>